doc/encoders.texi

```diff
@@ -3630,6 +3630,10 @@ performance. This option doesn't work if driver doesn't implement vaSyncBuffer
 function. Please make sure there are enough hw_frames allocated if a large
 number of async_depth is used.
 
+@item max_frame_size
+Set the allowed max size in bytes for each frame. If the frame size exceeds
+the limitation, encoder will adjust the QP value to control the frame size.
```

Comment:
You added a check for CQP in the new version, so it'd be better to add a description here.

Reply:
Added. thanks